react/week2/newtodo-list-app/src/ToDoListRow.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+
+const ToDoListRow = (props) => {
```

Comment:
If you want to make it more visible what the different props are for this component you can destructure the props object like so:
```suggestion
const ToDoListRow = ({ id, description, onRemove }) => {
```
In that case you can also remove the props. infront of id, description and onRemove further down. In general I would say it's easier to see which props are passed down when you destructure instead of just passing props, but it is a matter of taste.